dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -253,37 +253,35 @@ public Container getLiveContainerById(String id, User user, boolean respectFront
       return find(info.getLiveInode(), user, respectFrontendRoles);
 	}
 
-	@CloseDBIfOpened
-	@Override
-	@SuppressWarnings("unchecked")
-	public List<Container> getContainersInTemplate(final Template parentTemplate)
-			throws DotStateException, DotDataException, DotSecurityException {
 
-		List<Identifier> identifiers = new ArrayList<>();
-		DotConnect dc = new DotConnect();
-		dc.setSQL(
-				"select template_containers_2_.* from template_containers, identifier template_containers_1_,identifier template_containers_2_ "
-						+
-						"where template_containers.template_id = template_containers_1_.id and " +
-						"template_containers.container_id = template_containers_2_.id " +
-						"and template_containers.template_id = ? ");
-		dc.addParam(parentTemplate.getIdentifier());
-
-		try{
-			identifiers = ConvertToPOJOUtil.convertDotConnectMapToIdentifier(dc.loadResults());
-		}catch(ParseException e){
-			throw new DotDataException(e);
-		}
 
-		final List<Container> containers = new ArrayList<>();
-		for (Identifier id : identifiers) {
-			final Container cont = (Container) APILocator.getVersionableAPI()
-					.findWorkingVersion(id, APILocator.getUserAPI().getSystemUser(), false);
-			containers.add(cont);
-		}
-		return containers;
-	}
+    @CloseDBIfOpened
+    @Override
+    @SuppressWarnings("unchecked")
+    public List<Container> getContainersOnPage(final IHTMLPage page)
+            throws DotStateException, DotDataException, DotSecurityException {
+
+        final List<Container> containers = new ArrayList<>();
+        List<Identifier> identifiers = new ArrayList<>();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "identifiers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)